packages/request-pre/src/util/mock.ts

```
@@ -0,0 +1,13 @@
+import { ApiSchema } from '../api';
+import { requestReturn } from '../util/request';
+export default function addMock(requestInfo: ApiSchema): requestReturn | void {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)